### ISSUE TITLE
Loss function of UNetPlusPlus

### ISSUE THREAD

**Issue by leejm1031** [qubvel/segmentation_models.pytorch#312]
In Reference https://arxiv.org/pdf/1807.10165.pdf , Figure.1 Red Line -  "Deep supervision".  - Multiple input loss functions
Is Deep supervision  Implemented In this Package?

**Comment by JulienMaille** [comment]
Interesting, I guess the first step is to have Unet++ return all 4 layers and then feed them to the loss function?
![image](https://user-images.githubusercontent.com/182520/102191976-6f796d00-3eba-11eb-91aa-593dead3bf3f.png)

**Comment by leejm1031** [comment]
I'll do that. thanks!

**Comment by bhaktatejas922** [comment]
@leejm1031 @JulienMaille would you mind sharing what isn't achieved by smp already and what is achieved with this fix? For people like me who don't know at first glance. Appreciate it.

**Comment by azkalot1** [comment]
I will see to that - maybe we can add a way to return multiple masks. it is going to be useful for many models, not just Unet++

**Comment by JulienMaille** [comment]
@bhaktatejas922 I'm sorry I did not went any further, it is beyond my knowledge unfortunately

**Comment by pranshu97** [comment]
Are there any plans to implement this in the package?
It'd be nice to have a parameter in UnetPlusPlus like "mode = fast/accurate" where accurate mode gives avg from all 4 o/p nodes and fast mode from only the main o/p node as in the paper.
